debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleSnapshotChangeEventSource.java

```diff
@@ -122,12 +123,12 @@ protected void determineSnapshotOffset(RelationalSnapshotContext ctx) throws Exc
 
         ctx.offset = OracleOffsetContext.create()
                 .logicalName(connectorConfig)
-                .scn(currentScn)
+                .scn(currentScn.longValue())
```

Comment:
Why is the conversion to `long` needed?

Reply:
Mostly because the offset/source info changes were going to be done as a part of https://issues.redhat.com/browse/DBZ-2994.  What I could do is amend this PR with those changes as well and we have 1 full pull request that addressed the complete spectrum of these changes together.  Would that make more sense?

Reply:
No, it's fine as it already cptured elsewhere